scripts/street_sign_recognizer.py

```diff
@@ -43,22 +72,57 @@ def sign_bounding_box(self):
         (left_top, right_bottom) where left_top and right_bottom are tuples of (x_pixel, y_pixel)
             defining topleft and bottomright corners of the bounding box
         """
-        # TODO: YOUR SOLUTION HERE
-        left_top = (200, 200)
-        right_bottom = (400, 400)
+
+        # Convert colorspaces to HSV
+        cv_image_hsv = cv2.cvtColor(self.cv_image, cv2.COLOR_BGR2HSV)
+
+        # Apply filter to select only for objects in the yellow color spectrum
+        lb = np.round(np.multiply(self.hsv_lb, [255.0/360, 255.0/100, 255.0/100]))
+        ub = np.round(np.multiply(self.hsv_ub, [255.0/360, 255.0/100, 255.0/100]))
+        mask = cv2.inRange(cv_image_hsv, lb, ub)
+        self.cv_image_res = cv2.bitwise_and(self.cv_image, self.cv_image, mask = mask)
+
+        # Apply bounding box over most dense region
+        x, y, w, h = cv2.boundingRect(mask)
```

Comment:
I didn't realize you could do this without getting contours first, also, how do you know it is only over the most dense region? Is there a parameter you can tune for that?